### ISSUE TITLE
Global Widgets Spacing option is buggy

### ISSUE THREAD

**Issue by reardestani** [elementor/elementor#7309]
## Prerequisites
- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Global **Widgets Spacing** is acting differently in the latest version (2.5.1). In previous stable version it was fine. I am not sure it's a bug or new intended behavior but it does not look correct.

Video: http://take.ms/jwB7h

## Steps to reproduce
What above video.

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by gulhangulez** [comment]
Reza +1. Yes, I also faced with the same issue. https://monosnap.com/file/273jfu4SpRyEMlQkzLWa8Db5J1p3bz

**Comment by shilo-ey** [comment]
Hi @reardestani 

Thanks for reporting. we will look into this and update here.

Thanks.

**Comment by arielk** [comment]
Fix for v2.5.2

Thanks for your feedback

**Comment by mdrideout** [comment]
Using Version 2.5.9, just saved one of my pages, and then randomly all of my widgets across the site got a margin-bottom of 20px. Can't seem to get rid of it. 
```
.elementor-widget:not(:last-child) {
    margin-bottom: 20px;
}
```
